.ci/pipeline_definitions

```diff
@@ -46,7 +46,7 @@ dashboard:
         pull-request: ~
       steps:
         check:
-          image: 'node:20-alpine3.18'
+          image: 'node:20-alpine3.20'
```

Comment:
hm, seems like the renovate custom manager does not work in this case 🤔  https://github.com/gardener/dashboard/blob/c279ce4a1bf0ef6b5f4a324137d407d01bc891ec/.github/renovate.json5#L18-L24

Reply:
🤔

Reply:
There was also no update for the node image in the [Dockerfile](https://github.com/gardener/dashboard/blob/c279ce4a1bf0ef6b5f4a324137d407d01bc891ec/Dockerfile#L6)